Cargo.toml

```diff
@@ -79,96 +80,94 @@ members = [
     "crates/theme_selector",
     "crates/ui",
     "crates/util",
-    "crates/story",
-    "crates/vim",
     "crates/vcs_menu",
-    "crates/workspace",
+    "crates/vim",
     "crates/welcome",
+    "crates/workspace",
     "crates/zed",
     "crates/zed_actions",
 ]
 default-members = ["crates/zed"]
 resolver = "2"
 
 [workspace.dependencies]
-anyhow = { version = "1.0.57" }
```

Comment:
Any dependencies that just had a version number were updated to just use the `= "x.x.x"` syntax.